src/python/WMCore/WMException.py

```diff
@@ -11,7 +11,19 @@
 import logging
 import traceback
 import sys
+import re
 
+WMEXCEPTION_START_STR = "<@========== WMException Start ==========@>"
+WMEXCEPTION_END_STR = "<@---------- WMException End ----------@>"
```

Comment:
Can you add a new line to each of these 2 strings? Or it can cause problems when compiling this regex?
Otherwise you need to add a newline after L159 below.